src/SignhostAPIClient/Rest/DataObjects/Transaction.cs

```diff
@@ -7,9 +7,11 @@ public class Transaction
 	{
 		public string Id { get; set; }
 
+		public IDictionary<string, FileEntry> Files { get; set; }
```

Comment:
Both solutions would work. Also; the setter could be removed? Same counts for other readonly properties such as Transaction.Status?